src/main/java/Positive.java

```diff
@@ -0,0 +1,32 @@
+import java.util.regex.Pattern;
+
+public class Positive {
+
+    private static final Pattern NUMERIC_REGEX = Pattern.compile("-?\\d+");
+
+    private final String token;
```

Comment:
```suggestion
    private final int number;
```
양수 이기 때문에 이런 방식이 더 좋을거 같네요,
숫자 문자임과 양수임을 판단하는 부분은 생성자에 두고요 :)
어떤 장점이 있을지 한 번 고민해보세요